src/main/java/frc/robot/commands/PrepCommand.java

```diff
@@ -7,24 +7,33 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.Shooter;
 
-public class AmpCommand extends Command {
-  
+public class PrepCommand extends Command {
+ 
   private final Shooter shooter;
-  private double speed = 1; // TODO: Change to amp motor speed (in rotations per second)
-  private final double ampAlignAngle = 36; // TODO: tune relative to shooting from right in front of amp
+  double angle = 0;
+  double speed = 0;
 
-  public AmpCommand(Shooter shooter) {
+  /** Creates a new PrepCommand. */
+  public PrepCommand(Shooter shooter, double angle, double speed) {
+    // Use addRequirements() here to declare subsystem dependencies.
     this.shooter = shooter;
```

Comment:
This needs to have `this.angle = angle` and `this.speed = speed`. Otherwise when you call execute the `angle` and `speed` variables will still be 0.